### ISSUE TITLE
Add Gitlab connection support

### ISSUE THREAD

**Issue by ddattee** [ansible/galaxy#236]
I have my own gitlab instance and I would love to register my roles in the galaxy

**Comment by rainerduffner** [comment]
Related to this,I presume:

[https://github.com/ansible/galaxy/issues/160](url)

**Comment by ddattee** [comment]
Yes and no because I am talking about public gitlab repositories

**Comment by marcowork** [comment]
+1

**Comment by muchatlu** [comment]
+100

**Comment by julien-lecomte** [comment]
+1000

**Comment by muchatlu** [comment]
+1 , Is there any one Working on this?

**Comment by ironfroggy** [comment]
Content is published to Galaxy via [Collections](https://galaxy.ansible.com/docs/contributing/creating_collections.html), which are in preview as of Ansible 2.8, are published directly from your local machine and not imported from any repository into Galaxy, so you'll no longer need content hosted exclusively at Github.

**Comment by daniel-wtd** [comment]
Is there a proper documentation how one can automate the collection upload?
